CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,5 @@
+- Add: allow receive single HTTP attribute measures posted to `/iot/json/attrs/<attributeName>` with different content-types `application/json`, `text/plain` and `application/octet-stream`
+- Upgrade body-parser dep from 1.18.3 to 1.20.0
+- Upgrade express dep from ~4.16.4 to 4.18.1
+- Fix: do not use json parse strict for incoming http measures, commands and configuration
```

Comment:
Issue #661 should be cited (not sure if in the "Add:" line, in the "Fix:" line or in both)